scripts/install_linux.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Is this used? There is value in an example for automating install with web installer on linux, even if apt/yum/dnf is usually preferred for CI.

Reply:
I have such example in my fork and it uses this script. I'll transfer the example here.

Reply:
I added the Linux installer example that uses this script.